iosApp/iosApp/ui/pages/LandingScreen.swift

```diff
@@ -43,30 +65,24 @@ struct HeaderComponent: View {
     }
 }
 
-struct Buttons: View {
-    var navigation: NavigationController
-    
-    var body: some View {
-        VStack {
-            Button("Login") {
-                navigation.navigateTo(.login)
-            }
-            .frame(maxWidth: .infinity, maxHeight: 48)
-            .padding(.horizontal, 18)
-            .foregroundStyle(Color.blackPearl)
-            .background(Color.acid)
-            .cornerRadius(10)
-            Spacer().frame(height: 16)
-            Button("Registreren") {
-                navigation.navigateTo(.register)
-            }
-            .frame(maxWidth: /*@START_MENU_TOKEN@*/.infinity/*@END_MENU_TOKEN@*/, maxHeight: 48)
-            .padding(.horizontal, 18)
-            .foregroundStyle(.white)
-            .background(Color.smoke)
-            .cornerRadius(10)
-            .padding(.bottom, 16)
-        }
-        .padding(.bottom, 50)
-    }
-}
+//struct Buttons: View {
```

Comment:
Is this still needed?

Reply:
No, I commented in case I still needed it for styling or functionality.